Generate.py

```diff
@@ -15,10 +15,11 @@
 import Utils
 from worlds.alttp import Options as LttPOptions
 from worlds.generic import PlandoConnection
-from Utils import parse_yaml, version_tuple, __version__, tuplize_version, get_options, local_path, user_path
+from Utils import parse_yamls, version_tuple, __version__, tuplize_version, get_options, local_path, user_path
 from worlds.alttp.EntranceRandomizer import parse_arguments
 from Main import main as ERmain
 from BaseClasses import seeddigits, get_seed
+from typing import Set, Dict, List, Tuple
```

Comment:
you now have ìmport typing` as well as `from typing import x`. While it's functional to have both it's probably good to remove one form.  the from typing import should also be near the top with the other standard module imports. Generally you import standard lib, requirements, local project - in that order.